garak/cli.py

```diff
@@ -176,7 +176,7 @@ def main(arguments=None) -> None:
         "-d",
         type=str,
         default=_config.plugins.detector_spec,
-        help="list of detectors to use, or 'all' for all. Default is to use the probe's suggestion.",
+        help="list of detectors to use, or 'all' for all. Default is to use the probe's suggestion. For filtering with --list_detectors, use e.g. '--list_detectors -d misleading.Invalid'.",
```

Comment:
Please revert this line, the combination is only in effect if `--list_probes` or `--list_detectors` is used, the comment about combination should only be on those options.